documentation/_puppetserver_nav.html

```diff
@@ -6,58 +6,82 @@ <h3 id="puppetserver-{{server_version}}-docs">Puppet Server {{ server_version }}
+          <li><a href="/puppetserver/{{ server_version }}/external_ssl_termination.html">External SSL germination</a></li>
+          <li><a href="/puppetserver/{{ server_version }}/restarting.html">Restarting Puppet server</a></li>
+        </ul>
       </li>
-      <li><a href="/puppetserver/{{ server_version }}/puppet_conf_setting_diffs.html">Differing Behavior in puppet.conf</a></li>
-      <li><a href="/puppetserver/{{ server_version }}/gems.html">Using Ruby Gems</a></li>
-      <li><a href="/puppetserver/{{ server_version }}/subcommands.html">Subcommands</a></li>
-      <li><a href="/puppetserver/{{ server_version }}/compatibility_with_puppet_agent.html">Backward Compatibility With Puppet 3 Agents</a></li>
-      <li><a href="/puppetserver/{{ server_version }}/config_file_auth_migration.html">Migrating to the HOCON auth.conf Format</a></li>
-      <li><a href="/puppetserver/{{ server_version }}/external_ca_configuration.html">Using an External CA</a></li>
-      <li><a href="/puppetserver/{{ server_version }}/intermediate_ca.html">Puppet as an Intermediate CA</a><li>
-      <li><a href="/puppetserver/{{ server_version }}/external_ssl_termination.html">External SSL Termination</a></li>
-      <li><a href="/puppetserver/{{ server_version }}/tuning_guide.html">Tuning Guide</a></li>
-      <li><a href="/puppetserver/{{ server_version }}/restarting.html">Restarting the Server</a></li>
-      <li><strong>Known Issues and Workarounds</strong>
-        <ul>
-          <li><a href="/puppetserver/{{ server_version }}/known_issues.html">Known Issues</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/ssl_server_certificate_change_and_virtual_ips.html">SSL Problems With Load-Balanced PuppetDB Servers ("Server Certificate Change" error)</a></li>
+      <li><strong>Tuning, troubleshooting, and known issues</strong>
+        <ul>
+          <li><a href="/puppetserver/{{ server_version }}/known_issues.html">Known issues</a></li>
+          <li><a href="/puppetserver/{{ server_version }}/ssl_server_certificate_change_and_virtual_ips.html">SSL problems with load-balanced PuppetDB servers ("Server Certificate Change" error)</a></li>
+          <li><a href="/puppetserver/{{ server_version }}/tuning_guide.html">Puppet Server tuning guide</a></li>
+          <li><a href="/puppetserver/{{ server_version }}/compatibility_with_puppet_agent.html">Backward compatibility with Puppet 3 agents</a></li>
+          <li><a href="/puppetserver/{{ server_version }}/config_file_auth_migration.html">Migrating to the HOCON auth.conf format</a></li>
+        </ul>
+      <li><strong>Administrative API endpoints</strong>
+        <ul>
+          <li><a href="/puppetserver/{{ server_version }}/admin-api/v1/environment-cache.html">Environment cache</a></li>
+          <li><a href="/puppetserver/{{ server_version }}/admin-api/v1/jruby-pool.html">JRuby pool</a></li>
+        </ul>
+      </li>
+      <li><strong>CA v1 API endpoints</strong>
```

Comment:
Is there another version besides v1? If not, does it make sense to include `v1` at all?

Reply:
Yeah, none of the others have it, removing v1